js/main.js

```diff
@@ -1,12 +1,12 @@
-const TITLE = [
+const titlesAdt = [
```

Comment:
Почему после переименования эти сущности перестали быть константами?
Если под **adt** подразумевается **advertisement**, то понятнее использовать существительное: 
```suggestion
const AD_TITLES = [
```
Стоит избегать сокращений и аббревиатур в названиях, разработчики их могут трактовать по-своему или вовсе не понять, что усложнит работу с кодом. Если очень хочется, то можно использовать сокращения только [для расхожих случаев использования](https://www.kobzarev.com/programming/abbreviations-used-by-programmers/). 
По поводу нейминга [есть шутливая статья](https://learn.javascript.ru/ninja-code), может быть интересно.

Обрати внимание, что чекин и чекаут все еще в единственном числе.

-----

Более по-английски: **ROOM_TYPES**, **ROOM_FEATURES**, **ROOM_PHOTOS**, **AD_DESCRIPTIONS**.